packages/create-app/templates/default-app/packages/app/src/App.test.tsx

```diff
@@ -20,7 +20,10 @@ describe('App', () => {
       ] as any,
     };
 
-    const rendered = await renderWithEffects(<App />);
-    expect(rendered.baseElement).toBeInTheDocument();
+    const rendered = render(<App />);
```

Comment:
Couldn't find a good example test here that uses `screen` and that doesn't break as soon as you add a sign-in page